packages/lesswrong/lib/collections/users/tests.ts

```diff
@@ -5,16 +5,6 @@ testStartup();
 
 describe('updateUser – ', () => {
   let graphQLerrors = catchGraphQLErrors(beforeEach, afterEach);
```

Comment:
FWIW It seems better to me to change the test to update a user with a previous display name (so that the test is still validating an error gets thrown) instead of just dropping the test entirely.

Reply:
Yeah, that's fair.